src/WebJobs.Script/Diagnostics/ValueStopwatch.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.Azure.WebJobs.Script.Diagnostics
+{
+    // Based on: https://github.com/dotnet/aspnetcore/blob/master/src/Shared/ValueStopwatch/ValueStopwatch.cs
+    public struct ValueStopwatch
```

Comment:
Let's add some basic test coverage for this new type - would be good for it to have it's own test suite

Reply:
👍 - this may exist already at the source and reusable, either way will get this added

Reply:
Brought in the runtime tests and added one relevant to some of our usages.